ccm_web/server/src/api/api.section.handler.ts

```diff
@@ -96,7 +96,7 @@ export class SectionApiHandler {
         type
       }
     } catch (error) {
-      const errorResponse = handleAPIError(error, `Login ID: ${loginId}; Role: ${user.type}`)
+      const errorResponse = handleAPIError(error, `Login ID: ${loginId}; Role: ${user.role}`)
```

Comment:
Frankly, the user won't remember section id's than section name. The better option is to provide section name. That means is the the API call needs to send it as well and the type definitions needs to change `{loginID, role, sectionID, sectionName}`

Reply:
But the section ID is what is in the file? I don't think the section name makes sense here. Providing the section name in the input wouldn't do anything unless we allow functionality to update the section, which is not within scope.

Reply:
Since the enroll user has 2 workflows
1. For single section case, yes user selects section name and not ID. 
2. For multiple section case, user input sectionID but we show then a table to mapping out sectionId's and sectionName

If we just want to show sectionId's that I will suggest to show sectionID's in both cases. So that way expectation is equal. Sometime i forget what section I have selected. I don't mind doing the second way too. It is not that complex. LMK. I will work on it.

Reply:
Ok. I have added the section ID to Error handler. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/360/commits/a53a8509fa1891b4bfd35693ab92dc737f59b422.